packages/scene-composer/src/hooks/useAwsLightDarkModes.spec.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { createRef } from 'react';
+import { render } from '@testing-library/react';
+import { renderHook } from '@testing-library/react-hooks';
+const applyModeMock = jest.fn();
+jest.mock('@awsui/global-styles', () => ({
+  applyMode: applyModeMock,
+  Mode: jest.requireActual('@awsui/global-styles').Mode,
+}));
+import { Mode } from '@awsui/global-styles';
+
+import useAwsLightDarkModes from './useAwsLightDarkModes';
+
+describe('useAwsLightDarkModes', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('it should apply light mode', () => {
```

Comment:
NIT: should be changed to it('should apply light mode', ....
`it` in the quotes is redundant when you read the code

Reply:
fixed